e2e/src/authentication/login.e2e-spec.ts

```diff
@@ -0,0 +1,80 @@
+import { LoginPage } from './login.po';
+import { browser, logging, until } from 'protractor';
+
+describe('Login', () => {
+  let page: LoginPage;
+
+  beforeEach(() => {
+    page = new LoginPage();
+  });
+
+  describe('Given valid credential', () => {
+    it('enables form submit button', async () => {
+      await page.navigateTo();
+      await page.fillEmail('email@nimblehq.co');
+      await page.fillPassword('password');
+
+      expect(await (await page.getSubmitButton()).getAttribute('disabled')).toBeFalsy();
+    });
+
+    it('sets access token to local storage', async () => {
+      await page.navigateTo();
+      await page.LoginWith('dev@nimblehq.co', '12345678');
+
+      expect(localStorage.getItem('USER_ACCESS_TOKEN')).not.toBeNull();
+      expect(localStorage.getItem('USER_TOKEN_TOKEN')).not.toBeNull();
+    });
+
+    it('redirects to root url when hit login button', async () => {
```

Comment:
`hit` is not formal enough, it sounds too much like spoken English:

```suggestion
    it('redirects to the root url upon logging in', async () => {
```